external/anomaly/tests/test_ote_anomaly_classification.py

```diff
@@ -67,12 +74,12 @@ def test_cancel_training(self, task_path, template_path, dataset_path="./dataset
             category=category,
         )
         thread = Thread(target=self._trainer.train)
-        start_time = time.time()
         thread.start()
         self._trainer.cancel_training()
         thread.join()
-        # stopping process has to happen in less than 10 seconds
-        assert time.time() - start_time < 10
+
+        # If the training is cancelled model status cannot be success.
+        assert self._trainer.output_model.model_status != ModelStatus.SUCCESS
```

Comment:
where is  ModelStatus.SUCCESS set? Looks like if self._trainer.cancel_training() does nothing, this test passes